src/lib/structures/game/Game.ts

```diff
@@ -220,14 +230,27 @@ export abstract class Game {
 	 */
 	public abstract guess(guessMessage: Message): Promise<void>;
 
+	/**
+	 * @returns if the channel is where guesses are made for this game
+	 */
+	public abstract validGuessChannel(channel: TextChannel | DMChannel | PartialDMChannel): boolean;
+
 	/**
 	 * Handles game-specific behaviour for when the track ends.
 	 */
 	public abstract onTrackEnd(): Promise<void>;
 
+	/**
+	 * Calculates the points divisor for a player
+	 * Used in end()
+	 */
+	protected abstract calcPointsDivisor(songsListenedTo: number): number;
+
+	protected abstract getPlayers(vcMembers: Collection<string, GuildMember>): Collection<Snowflake, Player>;
```

Comment:
Same thing with dm, we'd have to capitalize initialisms
```suggestion
	protected abstract getPlayers(voiceChannelMembers: Collection<Snowflake, GuildMember>): Collection<Snowflake, Player>;
```